src/streetview/component.js

```diff
@@ -382,6 +400,7 @@ class StreetviewController {
 myModule.component('ngeoStreetview', {
   bindings: {
     'active': '<',
+    'configName': '@',
```

Comment:
```suggestion
    'optionsConstantName': '@',
```

Or `optionsName`
IMO configName is confusing.